pkg/services/alerting/extractor.go

```diff
@@ -203,16 +226,49 @@ func (e *DashAlertExtractor) getAlertFromPanels(jsonWithPanels *simplejson.Json,
 	return alerts, nil
 }
 
-func validateAlertRule(alert *models.Alert) bool {
-	return alert.ValidToSave()
+func validateAlertRule(alert *models.Alert) (ok bool, reason string) {
+	ok = alert.ValidToSave()
+	if !ok {
+		reason = fmt.Sprintf("Panel id is not correct, alertName=%v, panelId=%v", alert.Name, alert.PanelId)
```

Comment:
I think it's best to standardize on writing "ID" for identifier (id means [something else](https://en.wikipedia.org/wiki/Id,_ego_and_super-ego)):
```suggestion
		reason = fmt.Sprintf("Panel ID is not correct, alertName=%v, panelId=%v", alert.Name, alert.PanelId)
```